test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist.py

```diff
@@ -118,3 +125,38 @@ def _test_mnist_distributed(
         output = predictor.predict(data)
 
         assert output.shape == (batch_size, 10)
+
+        #  Check for Cloudwatch logs if true
+        if verify_logs:
+            _check_for_cloudwatch_logs(endpoint_name)
+
+def _check_for_cloudwatch_logs(endpoint_name):
+    client=boto3.client('logs')
+    response = client.describe_log_streams(
+        logGroupName='/aws/sagemaker/Endpoints/'+endpoint_name,    
+        orderBy='LastEventTime',
+        descending=True,
+        limit=5
+    )
+
+    logStreamName=response['logStreams'][0]['logStreamName']
+    print(logStreamName)
+
+    new_response = client.get_log_events(
+        logGroupName='/aws/sagemaker/Endpoints/'+endpoint_name,
+        logStreamName=logStreamName,
+        #startTime=int((datetime.today() - timedelta(days=2)).timestamp()),
+        #endTime=int(datetime.now().timestamp()),
+        limit=50,
+        startFromHead=True
+    )
+
+    print(new_response['events'])
+    recordsAvailable=bool(new_response['events'])
```

Comment:
this looks good for ensuring there are some logs. Now can we search for some specific messages that should be in the logs?

Reply:
Yes added logic for the same